.infrastructure/deployment.yml

```diff
@@ -27,6 +27,15 @@ spec:
           limits:
             memory: "256Mi"
             cpu: "150m"
+        volumeMounts:
+          - name: todoapp-data
+            mountPath: /app/data
+          - name: todoapp-config
+            mountPath: /app/configs
+            readOnly: true
+          - name: todoapp-secret
+            mountPath: /app/secrets
+            readOnly: true
```

Comment:
Ensure that the `ConfigMap` named `app-config` and the `Secret` named `app-secret` are created and available in the `todoapp` namespace. This is crucial for the application to access configuration and secret data.